config/cmake_ext_mod/ConfigureChecks.cmake

```diff
@@ -679,4 +679,3 @@ endif ()
 # the cache value is set in it's config file)
```

Comment:
# the cache value is set in its config file)

Reply:
????

Reply:
Fine, but I didn't change this on purpose. I will add it to one of my PRs on BB.